src/analyzer/analyzer.bif

```diff
@@ -49,3 +49,12 @@ function __tag%(name: string%) : Analyzer::Tag
 	analyzer::Tag t = zeek::analyzer_mgr->GetComponentTag(name->CheckString());
 	return t.AsVal();
 	%}
+
+function Analyzer::__set_ignore_checksums_nets%(v: any%) : bool
```

Comment:
hm - can we just use the correct type here? That might require adding subnet_set to init_bare.zeek - but that would enable type-checking by the script interpreter.